src/content/docs/2/guide/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+description: Stay tuned while we work on some delicious recipes.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
+## Setup
+
+First you have to install the notification plugin in your backend.
+
+<Tabs>
+    <TabItem label="Auto">
+    ```bash
+    tauri plugin add notification
+    ```
+    </TabItem>
```

Comment:
```suggestion
    <TabItem label="Automatic">
    
    Use the [add command in the Tauri CLI](/2/reference/cli/#plugin-add) to add the plugin:
    
    ```bash
    tauri plugin add notification
    ```
    </TabItem>
```

Reply:
I think we should list out all the npm/pnpm/cargo/etc. commands. Check out this component to see if it helps simplify that: https://github.com/tauri-apps/tauri-docs/blob/next/src/components/CommandTabs.astro

Reply:
Fixed